fe/src/views/homeView.jade

```diff
@@ -0,0 +1,14 @@
+doctype html
+html
+  head
+    meta(charset='UTF-8')
+    title Home
+    link(rel='stylesheet', type='text/css', href='base.css')
+  body
+    .main-container
+      .title Jukebox
+      .row
```

Comment:
I'm not sure if this structure is going to work. I was thinking something like this:
- **main-container** has 2 full-width children: **title** and **content** 
-  **title** has a relative height constraint (for example 10%)
- **content** fills the rest of the 90% height and has 2 children **widget** and **sliders**
- On desktop, **widget** and **sliders** are adjacent, and on mobile they're on top of one another